docker-compose.yml

```diff
@@ -6,5 +6,5 @@ services:
       REPORT_STREAM_URL_PREFIX: http://localhost:7071
     ports:
       - "8080:8080" # default api endpoint port
-      - "5005:5005" # Java debug port
+      - "6006:6006" # Java debug port
```

Comment:
RS runs the debug port on `5005`. This can create conflicts if TI uses the same port for debugging the container.